chia/wallet/dao_wallet/dao_wallet.py

```diff
@@ -1535,6 +1543,19 @@ async def generate_proposal_vote_spend(
                     get_innerpuzzle_from_cat_puzzle(Program.from_bytes(bytes(spend.puzzle_reveal)))
                 ).get_tree_hash()
             )
+        # vote_amounts_or_proposal_validator_hash  ; The qty of "votes" to add or subtract. ALWAYS POSITIVE.
+        # vote_info  ; vote_info is whether we are voting YES or NO. XXX rename vote_type?
+        # vote_coin_ids_or_proposal_timelock_length  ; this is either the coin ID we're taking a vote from
+        # previous_votes_or_pass_margin  ; this is the active votes of the lockup we're communicating with
+        # ; OR this is what percentage of the total votes must be YES - represented as an integer from 0 to 10,000 - typically this is set at 5100 (51%)
+        # lockup_innerpuzhashes_or_attendance_required  ; this is either the innerpuz of the locked up CAT we're taking a vote from OR
+        # ; the attendance required - the percentage of the current issuance which must have voted represented as 0 to 10,000 - this is announced by the treasury
+        # innerpuz_reveal  ; this is only added during the first vote
```

Comment:
Suggest clarifying comment:

>        # innerpuz_reveal  ; this is only added during the first vote

Does this mean that `innerpuz_reveal` is only passed in during the first vote, or announced, or both?